### PR TITLE
Improvements after review

### PR DESCRIPTION
**Pull request by Amuhar**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -46,9 +46,9 @@ import { StakingRouterModule } from '../staking-router-modules';
     }),
     ScheduleModule.forRoot(),
     KeyRegistryModule.forRootAsync({
-      inject: [SimpleFallbackJsonRpcBatchProvider],
-      async useFactory(provider) {
-        return { provider };
+      inject: [SimpleFallbackJsonRpcBatchProvider, ConfigService],
+      async useFactory(provider, configService) {
+        return { provider, keysBatchSize: configService?.get('KEYS_FETCH_BATCH_SIZE') };
```
Comment (avsetsin) [inline src/app/app.module.ts:51]: why `configService` is optional?

File: src/common/registry/test/testing.utils.ts [hunk 2]
```diff
@@ -35,13 +35,30 @@ export const compareTestMeta = async (
...
 
-// TODO: maybe add address as argument
-export const fetchKeyMock = (fromIndex = 0, toIndex = 1, expected: Array<any>) => {
-  return expected.splice(fromIndex, toIndex);
+export const clone = <T>(obj: T) => JSON.parse(JSON.stringify(obj)) as T;
+
+/** clears the db */
+// can we get rid of it?
+export const clearDb = async (orm) => {
+  // const orm = await MikroORM.init(mikroORMConfig);
```
Comment (avsetsin) [inline src/common/registry/test/testing.utils.ts:47]: unhelpful comments

File: src/http/sr-modules-validators/sr-modules-validators.service.ts [hunk 3]
```diff
@@ -110,6 +110,8 @@ export class SRModulesValidatorsService {
           percent: percent,
         });
 
+        // validator.exit_epoch === FAR_FUTURE_EPOCH
```
Comment (avsetsin) [inline src/http/sr-modules-validators/sr-modules-validators.service.ts:113]: ?

File: src/jobs/keys-update/keys-update.service.ts [hunk 4]
```diff
@@ -142,11 +137,13 @@ export class KeysUpdateService {
         for (const module of modules) {
           const moduleInstance = this.stakingRouterService.getStakingRouterModuleImpl(module.type);
 
-          // At the moment lets think that for all modules it is possible to make decision base on nonce value
+          // At the moment, let's assume that it is possible to make decisions for all modules based on the nonce value
```
Comment (avsetsin) [inline src/jobs/keys-update/keys-update.service.ts:140]: `nonce` should change in all modules with key changes:
https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/interfaces/IStakingModule.sol#L50-L61

so it's correct to update the keys when the nonce changes

**Review by avsetsin — APPROVED** [review]

File: src/app/app.module.ts (hunk 1 above)
Comment (Amuhar) [reply src/app/app.module.ts:51]: another way i get TypeError: Cannot read properties of undefined (reading 'get')
but like this it works, it is possible to set keys batch via this env

**Pull request merged**